app/2.0/docs/devguide/observers.md

```diff
@@ -100,15 +104,34 @@ The observer method is usually defined on the class itself, although an observer
 defined by a superclass, subclass, or a class mixin, as long as the named method exists on the
 element.
 
+Example 2: Provide a reference to an observer method { .caption }
+
+```js
+class XCustom extends Polymer.Element {
+
+  static get is() {return 'x-custom'; }
+
+  static get properties() {
+    return {
+      active: {
+        type: Boolean,
+        // Observer method
+        observer: 
+          function(newValue, oldValue){
+            this.toggleClass('highlight', newValue);
+          }.bind(this)
```

Comment:
I think a lambda reads clearer. Does the following work?

```js
observer(newValue, oldValue) {
  this.toggleClass('highlight', newValue);
}
```

Reply:
So this?

```
observer: 
  function observer(newValue, oldValue) {
    this.toggleClass('highlight', newValue);
  }.bind(this)
```

Or ...?

Reply:
Like this:

```js
static get properties() {
  return {
    active: {
      type: Boolean,
      // Observer method
      observer(newValue, oldValue) {
          this.toggleClass('highlight', newValue);
      }
  }
}
```

A running example: http://jsbin.com/xisifaboso/edit?html,console,output This is using the "object method shorthand definition" in ES2015: https://developer.mozilla.org/nl/docs/Web/JavaScript/Reference/Functions/Method_definitions

Reply:
Cool, didn't know that was a thing XD

Reply:
Yeah it is fairly new and neatly concise 😄  This PR now LGTM fully!

Reply:
Great, thanks!